### ISSUE TITLE
What is going to happen here?

### ISSUE THREAD

**Issue by Feliix42** [fsr/offline#1]
This repository lives here now for almost a year without anything happening. Will there be content in a foreseeable future or should this repository be deleted? :D

**Comment by kiliankoe** [comment]
Nah, don't delete it. Let it be a(nother) statement for @sjmielke's upcoming site fsrfails.de.

**Comment by Feliix42** [comment]
Doesn't seem like anything is going to happen here. So I'm going to delete this repository. We don't need the trash to pile up in the organization. 🗑 

Any objections?

**Comment by kiliankoe** [comment]
Objection! Same reason as last time.

**Comment by Feliix42** [comment]
Not accepted. Feel free to fork this in the next hours, before it's gone :wink:

**Comment by kiliankoe** [comment]
Is there any downside to having dead repos in the org? They're not even shown to anybody unless you explicitly tread into the depths not shown on the org's front page. 
The only downside I see is a possible nameclash if someone else wants to create a repo with the same name which then leads them to discover the old repo and any possible intentions of creating something before. I wouldn't have a problem with that.

**Comment by dirkonet** [comment]
Another objection. The Offline project will be resurrected soon™.

**Comment by LarsEngeln** [comment]
Offline is offline

**Comment by Feliix42** [comment]
♻️🗑 !

**Comment by kiliankoe** [comment]
Never!

**Comment by Feliix42** [comment]
@kiliankoe bist du da noch dran?

**Comment by kiliankoe** [comment]
Sowieso. Genauso wie vorher auch.
